packages/shared/dao/index.ts

```diff
@@ -198,11 +195,18 @@ export async function resolveRealmConfigFromString(realmString: string) {
 }
 
 export async function changeRealm(realmString: string, forceChange: boolean = false): Promise<void> {
+  const denylistedCatalysts: string[] = getDisabledCatalystConfig(store.getState()) ?? []
   const realmConfig = await resolveRealmConfigFromString(realmString)
 
   if (!realmConfig) {
     throw new Error(`The realm ${realmString} isn't available right now.`)
   }
+  const catalystURL = new URL(realmConfig.baseUrl)
+
+  if (denylistedCatalysts.find((denied) => new URL(denied).host === catalystURL.host)) {
+    commsLogger.log(`The realm is blacklisted.`)
+    return
```

Comment:
Throw?

Reply:
when I throw it's catch by the debugger, I though this is better

Reply:
The throw appears as a message in the chat, the log in the other hand only appears in the console